### PR TITLE
[general] Style cleanup with clang

### PR DESCRIPTION
**Pull request by jimmy-huang**
Signed-off-by: Jimmy Huang <jimmy.huang@intel.com>

### CODE REVIEW

File: src/jerry-port/zjs_jerry_port.c [hunk 1]
```diff
@@ -1,7 +1,7 @@
-// Copyright (c) 2017, Intel Corporation.
+// Copyright (c) 2017-2018, Intel Corporation.
 
-#include "jerryscript.h"
 #include "jerryscript-port.h"
+#include "jerryscript.h"
```
Comment (grgustaf) [inline src/jerry-port/zjs_jerry_port.c]: This one doesn't make sense, it was alphabetized better before. :)

Comment (jimmy-huang) [reply src/jerry-port/zjs_jerry_port.c]: yeah, somehow mine sort them with ../zjs before any alphabet and i didn't like that too, so i manually tweaked those, i'll tweak this one as well

**Comment by jimmy-huang** [comment]
@grgustaf patch updated

**Comment by grgustaf** [comment]
+1, but need to resolve conflicts

**Pull request merged**